apps/server/tsconfig.json

```diff
@@ -16,11 +16,19 @@
     "noImplicitAny": true,
     "strictBindCallApply": true,
     "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "paths": {
+      "@/*": ["src/*"],
+      "@common/*": ["src/common/*"],
+      "@config/*": ["src/config/*"],
+      "@decorator/*": ["src/decorator/*"],
+      "@filter/*": ["src/filter/*"]
```

Comment:
I think just `"@/*": ["src/*"]` would be enough.

Following would just use:
`@/common` for `src/common`, `@/config` for `src/config` and so on.

@SamipSthaLF Can you weigh in and verify?